### ISSUE TITLE
support dynamic line rewrapping

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#219]
```
Did you ever run a command whose output was formatted wider than your Mintty 
window? So, what do you do? Widen the window and re-issue the command. This is 
not always acceptable, for example if the command presents a snapshot of a 
time-based situation.

When the user changes the window size, why not re-wrap its contents, like 
Notepad does? If you think that that is unprecedented for a terminal emulator, 
check out Mac OS X's Terminal application. It's an excellent feature that ought 
to be the standard in terminal emulation packages.

--Ken Nellis
```

Original issue reported on code.google.com by `nelli...@gmail.com` on 14 Sep 2010 at 4:25
- Merged into: #82

**Comment by mintty** [comment]
Released 3.6.0.

**Comment by mintty** [comment]
Released 3.6.1.
